DataFormats/HcalRecHit/interface/HBHEChannelInfo.h

```diff
@@ -19,11 +19,11 @@ class HBHEChannelInfo
     static const unsigned MAXSAMPLES = 10;
 
     inline HBHEChannelInfo()
-      : rawCharge_{0.}, pedestal_{0.}, pedestalWidth_{0.}, gain_{0.}, riseTime_{0.f}, adc_{0},
+      : rawCharge_{0.}, pedestal_{0.}, pedestalWidth_{0.}, gain_{0.}, gainWidth_{0.}, riseTime_{0.f}, adc_{0},
```

Comment:
@slava77 the darkCurrent_, fcByPE_, and lambda_ are now scalars and not vector, the inizialization is done on a few lines below inside the clear() method